certs/certs.go

```diff
@@ -30,8 +30,18 @@ import (
 )
 
 const (
-	rsaKeySize   = 2048
 	duration365d = time.Hour * 24 * 365
+	rsaKeySize   = 2048
+	tlsKey       = "tls.key"
+	tlsCrt       = "tls.crt"
```

Comment:
The above constants are also defined upstream, if you want to point at those instead

Reply:
We should maybe remove them from CAPI and only have them here?

Reply:
I'm going to use the ones in this repo for the time being

Reply:
They're currently used in CAPI

Reply:
They're showing up unused in my editor (at least nothing in CAPI references them)

Reply:
I would expect CAPI to use tlsCrt with the kubeconfig generation...  it wouldn't need tlsKey

Reply:
Are you on latest master? The first two are used to create the certs, the TLSkey/crt are used to retrieve the Cluster CA

Reply:
I see them in use now that I pulled